src/fetch.js

```diff
@@ -24,13 +24,44 @@ module.exports = async (entityDefinition, ctx) => {
   );
 
   try {
+    if (typeof loop === 'number') {
+      return fetchLoop(reporter, queryLimit, requestOptions, loop);
+    }
+
     const { data } = await axios(requestOptions);
     return castArray(data).map(clean);
   } catch (error) {
     reporter.panic(`Failed to fetch data from Strapi`, error);
   }
 };
 
+const fetchLoop = async (
+  reporter,
+  queryLimit,
+  requestOptions,
+  loop,
+  resultsAggreg = [],
+  _start = 0
```

Comment:
Can you replace these arguments with an object? This way it will be easier to read when calling that function